effectsize.Rmd

```diff
@@ -0,0 +1,99 @@
+# Effect size {#effectsize}
+<div style="text-align:center">
+```{r}
+knitr::include_graphics("figures/effectsize/teaser.png")
```

Comment:
Coicidentally I also came across this today, which looks like it has a nicer interface than gtable: https://cran.r-project.org/web/packages/cowplot/vignettes/introduction.html